src/System.Linq/tests/System.Linq.Tests.csproj

```diff
@@ -36,5 +37,10 @@
       <Name>System.Linq</Name>
     </ProjectReference>
   </ItemGroup>
+  <ItemGroup>
```

Comment:
This shouldn't be necessary... it should happen implicitly due to having added the package entry to packages.config.  In fact, I'm a bit surprised having it here as well isn't causing a warning.